nogo_config.json

```diff
@@ -125,5 +125,14 @@
       "vendor/": "vendor doesn't pass vet",
       "external/": "externaldoesn't pass vet"
     }
+  },
+  "errcheck": {
+    "exclude_files": {
+      "vendor/": "vendor doesn't pass errcheck",
+      "external/": "external doesn't pass errcheck",
+      "src/": "golang core code does not pass errcheck",
+      "pkg/": "KubeVirt pkg does not pass errcheck yet",
+      "cmd": "KubeVirt binaries do not pass errcheck yet"
```

Comment:
My intention is for follow-up PRs to individually remove these. 

Didn't want to overload this PR.